### ISSUE TITLE
Add collapse menu for Login/logout/dashboard

### ISSUE THREAD

**Issue by Shriukan33** [transport-nantes/tn_web#603]
The only way to access those pages is a direct url, you can't get there from homepage.

**Comment by JeffAbrahamson** [comment]
I don't think we need a specific login button (yet), but if the user is logged in, we might add some extra items to the menu.
